cg/meta/rsync.py

```diff
@@ -38,17 +38,26 @@ def run_rsync_command(self, ticket_id: int = None, dry_run: bool = False) -> Non
             raise CgError()
 
         customer_id: str = cases[0].customer.internal_id
-        rsync_destination_path = self.destination_path % (customer_id, ticket_id)
-        delivery_source_path = (
+        rsync_destination_path: str = self.destination_path % (customer_id, ticket_id)
+        delivery_source_path: str = (
             Path(self.delivery_path, customer_id, "inbox", str(ticket_id)).as_posix() + "/"
         )
         rsync_options = "-rvL"
-        parameters = [rsync_options, delivery_source_path, rsync_destination_path]
+        parameters: List[str] = [rsync_options, delivery_source_path, rsync_destination_path]
         self.process.run_command(parameters=parameters, dry_run=dry_run)
 
         if cases[0].data_analysis == Pipeline.SARS_COV_2:
             LOG.info("Delivering report for SARS-COV-2 analysis")
-            covid_report_path = self.covid_report_path % (str(cases[0].internal_id), ticket_id)
-            covid_destination_path = self.covid_destination_path % customer_id
-            parameters = [rsync_options, covid_report_path, covid_destination_path]
+            covid_report_options: List[str] = glob.glob(
```

Comment:
Ok I suggested to let `self.covid_report_path` be a `Path`-object and then use the `pathlib.Path.glob()` functionality. Feel free to do as you like. I think it is more nice to work with pathlib paths whenever possible